test/Scripts.Integration.Test/Scripts/CliConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Sentry.Unity;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+public class CliConfiguration : SentryCliOptionsConfiguration
+{
+    public override void Configure(SentryCliOptions cliOptions)
+    {
+        Debug.Log("Sentry: CliConfiguration::Configure() called");
+
+        // Assert the deprecated options have been set and are getting overwritten
+        // Assert.AreEqual("Old configure got called", cliOptions.Auth);
```

Comment:
Not asserting this? should we?

It's important to verify we're not breaking people who are upgrading btw. 

Without snapshot testing we're just going from a blank state, so we won't validate that

Reply:
Good catch. This is dead code and actually the other way around. First New than Old.